cmd/preupgradechecks/preupgradechecks.go

```diff
@@ -87,7 +87,7 @@ func (client *PreUpgradeTaskClient) LatestSchemaApplied() error {
 		return errors.New("Could not get the Function CRD")
 	}
 	// Any new field added in Function spec can be checked here provided the substring matches the description in CRD Validation of the field
-	if !strings.Contains(crd.Spec.String(), "RequestsPerPod") {
+	if !strings.Contains(crd.Spec.String(), "RequestsPerPod") && !strings.Contains(crd.Spec.String(), "OnceOnly") {
```

Comment:
Should this be `||` instread of `&&` ?

Reply:
We want to be sure both are present since we're introducing both in one release.